external/qunit/qunit.js

```diff
@@ -1172,7 +1172,7 @@ Test.prototype = {
 			filter = filter.slice( 1 );
 		}
 
-		// If the filter matches, we need to honour include
+		// If the filter matches, we need to honor include
```

Comment:
anything in `/external` is an external dependency so you'd need to report it in that project.